<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShipmentForm</class>
 <widget class="QWidget" name="ShipmentForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Схема-расчет выдачи</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../transport_control.qrc">
    <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Распоряжение:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbOrder">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tvShipment"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbSave">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCancel">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_exit">
         <property name="text">
          <string>Выход</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbOrder</tabstop>
  <tabstop>pbSave</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>pushButton_exit</tabstop>
  <tabstop>tvShipment</tabstop>
 </tabstops>
 <resources>
  <include location="../transport_control.qrc"/>
 </resources>
 <connections/>
</ui>
